bin/otomi

```diff
@@ -130,11 +131,10 @@ function evaluate_secrets() {
     [ -n "$VERBOSE" ] && echo "Info: The 'secrets.*.yaml files' are not decrypted, because $ENV_DIR/.sops.yaml file is not present"
     return 0
   }
-  if [ -z "$CI" ]; then
-    local secrets_path="$ENV_DIR/.secrets"
-    [ ! -r "$secrets_path" ] && err "Unable to find the '$secrets_path' file.\nPlease follow to documentation: $readme_url" && exit 2
-    source $secrets_path
-  fi
+  local secrets_path="$ENV_DIR/.secrets"
+  [ ! -r "$secrets_path" ] && err "Unable to find the '$secrets_path' file.\nPlease follow to documentation: $readme_url" && exit 2
+  # shellcheck source=/dev/null
+  . $secrets_path
```

Comment:
The logic introduced here wasn't tested. I can understand that it's difficult to make changes to our bash plumbing without a test suite but **this logic puts the carriage before the horse.** 

I don't understand why it would only source secrets when in CI, because it is also a dependency for some commands that think `.secrets` is there. An enhancement would be to check if `.secrets` are present but I just put some bandages on the problem for now.

Reply:
Why did you remove the check for CI?? It is there for a reason: if not in CI then source the secrets

Reply:
Please add it back but check for 'true'